openml-h2o/src/main/java/com/feedzai/openml/h2o/H2OFeatureImportanceCreator.java

```diff
@@ -97,7 +97,7 @@ public VarImp calculateFeatureImportance(final Dataset dataset,
             final Path datasetPath = H2OUtils.writeDatasetToDisk(dataset);
             final Model model = this.h2OApp.train(gbm.getAlgorithmDescriptor(), datasetPath, dataset.getSchema(), params, random.nextLong());
             // safe cast as long as using H2OAlgorithm.GRADIENT_BOOSTING_MACHINE
-            return ((GBMModel) model)._output._varimp;
+            return model._output._varimp;
```

Comment:
don't modify h2o, put this on a separate MR if need be

Reply:
done